style.css

```diff
@@ -128,3 +129,279 @@ body {
     margin-left: 8rem;
   }
 }
+
+.container-work {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto fit);
+  place-content: left;
+  gap: 1rem;
+  margin-left: 25px;
+  height: 40vh;
+}
+
+.card-info {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  gap: 1rem;
+  overflow: visible;
+}
+
+.card-image {
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  border-radius: 5px;
+  margin-left: 2px;
+  box-shadow: 0 5px 10px -3px rgba(green, blue, alpha);
+}
+
+.card-image > img {
+  border-top-right-radius: 5px;
+  border-bottom-left-radius: 5px;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 350px;
+  height: 240px;
+}
+
+.card-header {
+  font-size: 20px;
+  font-weight: 700;
+  display: flex;
+  flex-direction: column;
+  justify-content: right;
+  color: #000;
+  line-height: 1.5;
+  letter-spacing: 1.8;
+}
+
+.card-details {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+  font-size: 16px;
+}
+
+.card-details > span:nth-child(1) {
+  text-transform: uppercase;
+  color: #505f79;
+  font-weight: 700;
+}
+
+.card-details > span:nth-child(3) {
+  color: lightslategray;
+  font-weight: 600;
+}
+
+.card-details > span:nth-child(5) {
+  color: lightslategray;
+  font-weight: 600;
+  font-size: 18px;
+}
+
+.circle-icon {
+  transform: scale(0.5);
+  color: #505f79;
+}
+
+.card-content {
+  font-size: 16px;
+  font-weight: 400;
+  color: #344563;
+  max-width: 350px;
+}
+
+.btn-languages {
+  background-color: rgba(135, 148, 250, 0.016);
+  border: none;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  width: 6rem;
+  height: 2rem;
+  border-radius: 5px;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 3px 5px;
+}
+
+.btn-languages:hover {
+  background-color: #fff;
+}
+
+.btn-link {
+  text-decoration: none;
+  color: #7f8cff;
+}
+
+.btn-link:hover {
+  color: aqua;
+  cursor: pointer;
+}
+
+.btn-card {
+  background-color: #fff;
+  border: 1px solid #7f8cff;
+  padding: 14px 20px;
+  color: #7f8cff;
+  text-align: center;
+  text-decoration: none;
+  font-size: 16px;
+  width: 8.5rem;
+  height: 3rem;
+  display: flex;
+  margin-left: 8px;
+  border-radius: 12px;
+  font-weight: 700;
+  margin-bottom: 2rem;
+}
+
+.btn-card:hover {
+  cursor: pointer;
+  background-color: #505f79;
+}
+
+#about_me {
+  margin: 150px 0;
+  width: 100%;
+  min-height: 1701px;
+  color: #505f79;
+}
```

Comment:
As CSS best practices, let's use class to style element instead of id for specificity

kindly follow the Figma template design as required including font, spacing, and color
![image](https://user-images.githubusercontent.com/66112760/164888017-a300595c-9167-442e-ac36-2a1f32a2804b.png)